akka-cluster-sharding-typed/src/test/scala/docs/akka/cluster/sharding/typed/AccountExampleUnpersistentDocSpec.scala

```diff
@@ -39,40 +33,28 @@ class AccountExampleUnpersistentDocSpec
         //  protocol
         snapshotProbe.hasEffects shouldBe false
 
-        testkit.run(AccountEntity.GetBalance(getBalanceInbox.ref))
-
-        getBalanceInbox.receiveMessage().balance shouldBe 0
+        testkit.ask[AccountEntity.CurrentBalance](AccountEntity.GetBalance(_)).receiveReply().balance shouldBe 0
       }
     }
 
     "handle Deposit and Withdraw" in {
-      val replyToInbox = TestInbox[StatusReply[Done]]()
-      val getBalanceInbox = TestInbox[AccountEntity.CurrentBalance]()
-
       onAnOpenedAccount { (testkit, eventProbe, _) =>
-        testkit.run(AccountEntity.Deposit(100, replyToInbox.ref))
+        testkit.askWithStatus[Done](AccountEntity.Deposit(100, _)).expectDone()
 
-        replyToInbox.expectMessage(StatusReply.Ack)
         eventProbe.expectPersisted(AccountEntity.Deposited(100))
 
-        testkit.run(AccountEntity.Withdraw(10, replyToInbox.ref))
+        testkit.askWithStatus[Done](AccountEntity.Withdraw(10, _)).expectDone()
```

Comment:
Wonder if we should name it `runAskWithStatus` and `runAsk`?
There is probably a good reason for that it is named `run` and not `tell`.

Reply:
I kind of interpreted `run` vs. `tell` as denoting the synchronous nature.  `ask` suggests some more synchronicity than `tell`.

`runWithReply`/`runForReply`/`runExpectingReply`(???) (s/Reply/StatusReply/) as appropriate also sound good in that direction to my ear.

Reply:
`runAsk` has the advantage of associating with the ask pattern, though

Reply:
Good. I think runAsk is better (even though it isn't runTell)